src/views/governance/components/AccountVotes.tsx

```diff
@@ -53,6 +56,10 @@ const AccountVotes = () => {
           <SmallButton mt={3} variant="muted" onClick={handleDelegate}>
             <Trans>Delegate</Trans>
           </SmallButton>
+          <Text ml={3} variant="legend" as="span">
+            <Trans>Voting Power: </Trans>
```

Comment:
We should change this to "Balance" or other label, in case the account hasn't delegate or delegated to a different account this label would be inaccurate